src/ontology/mondo-edit.obo

```diff
@@ -617334,6 +617334,19 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-7638-4659
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9001" xsd:anyURI
 
+[Term]
+id: MONDO:1060120
+name: ACO2-related optic atrophy with or without extraocular features
+def: "Any primary mitochondrial disease in which the cause of the disease is monoallelic or biallelic variants in the ACO2 gene. While optic atrophy is present in most affected cases, ACO2 is a mitochondrial protein and thus features of this disease include abnormal mitochondrial morphology and can affect other organ systems. Extraocular features can include ataxia, spastic paraplegia, CNS abnormalities, neurodevelopmental phenotypes, and retinal degeneration." [PMID:29564393]
```

Comment:
this definition is not consistent with the label. Please ask them if this should be a type of optic atrophy or if it is a mitochondrial disease, the label would make more sense  as ACO2-related mitochondrial disease, where optic atrophy is a feature of the disease.